catalog-info.yaml

```diff
@@ -36,6 +39,46 @@ spec:
         SLACK_NOTIFICATIONS_ALL_BRANCHES: "false"
         SLACK_NOTIFICATIONS_ON_SUCCESS: "false"
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-docs-pr
+  description: Build the docs pull requests
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/docs-build-pr
+
+spec:
+  type: buildkite-pipeline
+  owner: group:docs
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: docs / build-pr
+    spec:
+      repository: elastic/docs
+      pipeline_file: ".buildkite/build_pr_pipeline.yml"
+      provider_settings:
+        trigger_mode: none # triggered via webhook and the buildkite-pr-bot
+        skip_intermediate_builds: false
+      teams:
+        docs-build-guild:
+          access_level: MANAGE_BUILD_AND_READ
+        ci-docs-migration-taskforce:
+          access_level: MANAGE_BUILD_AND_READ
+        docs:
+          access_level: BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
+      env:
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: "true"
```

Comment:
I don't know if we want Slack notifications when a build on a PR fails. That's up to the docs team, though.